docs/UserGuide.adoc

```diff
@@ -56,85 +66,128 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
+* `edit 1 f/Halo t/BahasaIndonesia` +
+Flashcard at index 1 will be edited and the updated information will be displayed
 
-=== Locating persons by name: `find`
+=== Locating a Flashcard by keyword: `find`
 
-Finds persons whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Finds flashcards whose names contain any of the given keywords. +
+
+Format: ​`find [KEYWORD]... [f/KEYWORD]... [b/KEYWORD]... [t/KEYWORD]...`
 
 ****
-* The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Finds the flashcard(s) that contains the given keyword(s)
 ****
 
 Examples:
 
-* `find John` +
-Returns `john` and `John Doe`
-* `find Betsy Tim John` +
-Returns any person having names `Betsy`, `Tim`, or `John`
+* `find f/你好 t/language` +
+This will display flashcards that contains the given keyword in language tag
+* `find t/spanish` +
+This will display all flashcards with spanish tag
+* `find` +
+This will display all flashcards
+
+=== Deleting a flashcard : `delete`
 
-=== Deleting a person : `delete`
+Deletes an existing Flashcard. +
 
-Deletes the specified person from the address book. +
 Format: `delete INDEX`
 
 ****
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+* Deletes the flashcard at the specified INDEX.
+* The index refers to the index number shown in the list all flashcards command.
 * The index *must be a positive integer* 1, 2, 3, ...
 ****
 
 Examples:
 
-* `list` +
+* `find` +
 `delete 2` +
-Deletes the 2nd person in the address book.
-* `find Betsy` +
+Deletes the 2nd flashcard in the list.
+* `find f/Betsy` +
 `delete 1` +
-Deletes the 1st person in the results of the `find` command.
+Deletes the 1st flashcard in the results of the `find` command.
+
+=== Quiz mode : ​`quiz`
+
+Enter quiz mode. A card will be shown to the user one by one randomly with distribution based
+on accuracy of each flashcard. The user then can self-verify whether he/she correctly guess the other side of the flashcard.
+
+Format: `​quiz [t/TAG] []`
+
+****
+* Enters quiz mode. If no tags are given, all cards can be shown.
+****
+
+Examples:
+
+* `quiz t/chinese`
+
+=== Show progress statistics : s​tats
+Show statistics for the cards with the given tag, or all cards if tag is not specified, including
+accuracy, number of tries, etc.
+
+Format: ​stats [t/TAG]
+
+****
+* Shows the statistics of the cards with the given tag. If no tags are given, statistics of all cards will be shown.
+****
+
+Examples:
+
+* `stats t/physics`
+
 
-=== Selecting a person : `select`
+=== [Coming in version 2.0] Selecting a person : `select`
```

Comment:
@lungsin Make this and others "[Coming in v2.0]"? (to make it short)

Reply:
Updated